CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/ConfigurationPane.java

```diff
@@ -240,9 +219,9 @@ void update(final List<ImportDefinition> definitions) {
         // (This tends to involve Platform.runLater() so let them be queued.)
         tabPane.getTabs().clear();
 
-        for (final ImportDefinition impdef : definitions) {
+        definitions.forEach(_item -> {
```

Comment:
would be good to rename the parameter such that there is no underscore (following naming convention)